### DISCUSSION TITLE
How can I be a github community maintainer

### DISCUSSION THREAD

**Discussion by eymeen** [community/community#118693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm interested in being a moderator/maintainer, is GitHub welcoming requests and are there any rules to be one?

**Comment by ebndev** [comment]
Hi @eymeen,

Thank you for being a part of the GitHub community and for all of your contributions!

All of the moderators and maintainers of this forum are GitHub staff members, not volunteer based as you may find in other communities on Discord (for example).

As this is a public forum and driven by community members like yourself, I highly encourage you continue engaging in discussions as you have been! You can also report behavior and content that violates community guidelines and terms by referring to the links below.

The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

Thank you!
